test/gurka/test_osrm_serializer.cc

```diff
@@ -515,3 +515,196 @@ TEST(Standalone, HeadingNumberAutoRoute) {
   }
   // clang-format on
 }
+
+TEST(Standalone, BannerInstructions) {
```

Comment:
ausführlich, respekt!

Reply:
I realised later on that I could have used `assert_json_equality(serialized_json, expected_json)` instead, but this works too...